app/assets/stylesheets/components/_service-specific-info.scss

```diff
@@ -6,6 +6,20 @@
   .sidebar-accordion {
     margin: 0 0-$site-margins-mobile 2rem 0-$site-margins-mobile;
   }
+
+  .video_wrapper {
+    padding-top: 56.25%;
```

Comment:
56.25% = 315/560 (height/width aspect ratio)

Reply:
I'm a little confused by this. Does this mean that the iframe will be shifted down by 9/16 of its height?

Reply:
Since I can't dynamically size the iframe to have the 9/16 aspect ratio, this adds a wrapper around the iframe with padding-top (height) = 9/16 of width. The iframe is positioned absolutely at 0,0 in the wrapper so it's not shifted down.

Reply:
@jamesatheyDDS This is a version of the intrinsic ratio trick… https://alistapart.com/article/creating-intrinsic-ratios-for-video

Reply:
Oops, good thing we don't care about IE5 & 6? :)